client/state/selectors/test/get-past-billing-transaction.js

```diff
@@ -57,4 +58,22 @@ describe( 'getPastBillingTransaction()', () => {
 		);
 		expect( output ).to.be.null;
 	} );
+
+	test( 'should return a billing transaction that has been fetched individually', () => {
```

Comment:
Do we want to add some more tests for cases where transactions are not being fetched, or are missing?